source/changelog.txt

```diff
@@ -12,13 +12,32 @@
    :depth: 1
    :class: singlecol
 
+v1.3.0
+------
+
+*Released March 17, 2022*
+
+- :issue:`MONGOSH-856` - Kerberos feature parity with the legacy shell 
+  is Done™ now, with the last command line option now also working as it
```

Comment:
I'm not sure if you want to copy my fairly casual "Done™" into the release notes here, it might not fit the typical tone of official release notes, but in terms of content this is fine :)